### PR TITLE
feat: add transcription from browser and download SRT

### PR DESCRIPTION
**Pull request by GuiBibeau**
# Description:

TL/DR Allows you to upload your audio file and get a subtitle file

in details:

- Uses transformers.js in a web worker to load whisper (for now only whisper tiny so the performance is meh)
- it grabs the results from the model and pushes it into state
- allows you to download the subtitle file after it is ready

## To test:

1 go to /gen-subtitles
2. upload any audio file that originates from a video 
3. watch in the console, the web worker showing progress on the transcription
4. on screen the srt file is being shown (unbeautified for now)
5. download srt file and test it (rename to <video-name-here>.srt
6. play video in a modern player while video and srt are in same folder

### CODE REVIEW

**Review by tobySolutions — APPROVED** [review]
LGTM!!! This is awesome @GuiBibeau, I hope to be as good as this someday. The only things that I see left are minor touch-ups like you said. 


Also what's the webpack thingy?
